src/config/userTypes.ts

```diff
@@ -51,6 +51,8 @@ export interface IHacker {
     shortAnswer: {
       // array of mongoose ids referencing different skills
       skills?: string[];
+      // number of previously attended hackathons
+      previousHackathons: string;
```

Comment:
Why a string instead of a number?

Reply:
hey! yeah it was because one of the values was "5+", i have the previousHackathons field setup as a string in the backend as well, if needed, i can change it to a number

Reply:
I think doing this as an int will be easier to do stats and queries at the database level.

Since this data is never likely to be summed, there's no harm in conflating 5 with 5+ IMO. Though this does introduce a consistency issue that the backend will need a guard against ints > 5.

Reply:
Yeah. I feel like "5 or more" can be conflated with 5.

Reply:
Yeah, this makes sense

Reply:
alright sounds good! i'll make the changes in the backend first and then ping u guys, thx for the explanation